### DISCUSSION TITLE
I want to control the test unit

### DISCUSSION THREAD

**Discussion by suzu-yuta** [chaitu-ycr/py_canoe#58] [Q&A]
Hello!
I find py-CANoe very easy to use and it has been very helpful.

I have a question and a request: Is it possible to control test units generated in vTEAT Studio using py-CANoe? I tried, but it seems they are not recognized and only test modules generated in CAPL are accepted.

Do you have any plans to support test units in the future? Please let me know.

Thank you as always.

**Comment by chaitu-ycr** [comment]
hi @suzu-yuta 

executing vtest test configuration is available in py-canoe version 26.1.0
https://chaitu-ycr.github.io/[py-canoe](https://chaitu-ycr.github.io/py-canoe/#execute-test-configuration-test-units)/#execute-test-configuration-test-units
